eng/pipelines/dotnet-buildtools-prereqs-all-pr.yml

```diff
@@ -0,0 +1,25 @@
+trigger: none
```

Comment:
This file seems to be almost exactly the same as [eng/pipelines/dotnet-buildtools-prereqs-all.yml](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/732/files#diff-b18af6e0e549a23ecd4af3675df1948a2f64ff304c74dcaa9a23d7980e01f068). That pipeline even has conditions for steps that should only run on internal project, which leads me to believe it should be fine to run in the public. Would it make sense to add a PR trigger to that other pipeline and not add a new yaml?

Reply:
This is the current standard practice in this repo. Just trying to match everything else.

Reply:
If the duplication is desirable, this change looks good to me I guess. But having a 1:1 duplicated code pipeline doesn't sit all that well with me.

Reply:
The reason for this is related to the triggers and trying to avoid wasting compute.  See https://github.com/dotnet/dotnet-buildtools-prereqs-docker/issues/328 for the background.

Reply:
Hmm yeah, duplicating the YAML files seems to be a good workaround for that. The only other alternative I can think of is to change the triggers on the public pipeline through the pipeline UI, but that also has its drawbacks.